react-native/src/blueDarkTheme.ts

```diff
@@ -9,23 +9,25 @@ This code is licensed under the BSD-3 license found in the LICENSE file in the r
 import { configureFonts, DefaultTheme } from 'react-native-paper';
 import { blue, red, darkBlack, black, lightBlue } from '@pxblue/colors';
 import { fontConfig } from './shared';
-import { Theme } from 'react-native-paper/lib/typescript/src/types';
+import * as color from 'color';
```

Comment:
This is the typical way we import the color package (doesn't make a difference functionally, but for style consistency we may want to swap).
```suggestion
import Color from 'color';
```

Reply:
the uppercase kinda sorta helps distinguish from `theme.colors`